File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    if ('a' <= s[0] && s[0] <= 'z')
        answer += toupper(s[0]);
    else
        answer += s[0];
    for (int i = 1; i < s.size(); i++) {
        if (s[i - 1] == ' ' && s[i] != ' ')
            answer += toupper(s[i]);
        else
            answer += tolower(s[i]);
    }
    return answer;
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = []
    if 'a' <= s[0] <= 'z':
        answer.append(s[0].upper())
    else:
        answer.append(s[0])
    for i in range(1, len(s)):
        if '0' <= s[i] <= '9':
            answer.append(s[i])
        elif s[i - 1] == ' ' and s[i] != ' ':
            answer.append(s[i].upper())
        else:
            answer.append(s[i].lower())
    return ''.join(answer)